exporters/zipkin/CMakeLists.txt

```diff
@@ -37,6 +37,8 @@ install(
   PATTERN "recordable.h" EXCLUDE)
 
 if(BUILD_TESTING)
+  add_definitions(-DGTEST_LINKED_AS_SHARED_LIBRARY=1)
```

Comment:
So in windows, we only have gtest dll?

Reply:
There is some unresolved global lock variable in gtest/gmock reported if built without this flag.